src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -42,12 +42,13 @@ internal sealed partial class SolutionState
 
     private readonly SolutionInfo.SolutionAttributes _solutionAttributes;
     private readonly ImmutableDictionary<ProjectId, ProjectState> _projectIdToProjectStateMap;
-    private readonly FilePathToDocumentIdsMap _filePathToDocumentIdsMap;
     private readonly ProjectDependencyGraph _dependencyGraph;
 
     // holds on data calculated based on the AnalyzerReferences list
     private readonly Lazy<HostDiagnosticAnalyzers> _lazyAnalyzers;
 
+    private ImmutableDictionary<string, ImmutableArray<DocumentId>> _lazyFilePathToRelatedDocumentIds = ImmutableDictionary<string, ImmutableArray<DocumentId>>.Empty;
```

Comment:
>_lazyFilePathToRelatedDocumentIds [](http://example.com/codeflow?start=68&length=33)

Not sure what the "Related" part of this name means

Reply:
discussed offline.  used to respond to GetRelatedDocumentIds.